wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt.java.interop

```diff
@@ -25,7 +25,7 @@ data class Person(
     @field:WireField(tag = 3, adapter = "com.squareup.wire.ProtoAdapter.STRING") val email: String?
             = null,
     @field:WireField(tag = 4, adapter = "PhoneNumber.ADAPTER") val phone: List<PhoneNumber> =
-            emptyList(),
+            Internal.newMutableList(),
```

Comment:
Guess I know where this is coming from: the default value for a repeated field is calculated as
```kotlin
isRepeated -> {
  if (javaInterOp) {
    val internalClass = ClassName("com.squareup.wire.internal", "Internal")
    CodeBlock.of("%T.newMutableList()", internalClass)
  } else {
    CodeBlock.of("emptyList()")
  }
}
```
I think we should always use `emptyList()`, even in Java interop mode, Wire compiler will still depend on kotlin-stdlib, so that's fine. Could you please change that and regenerate the tests?